### PR TITLE
ydotool: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/ydotool.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ydotool
+
+> Display server agnostic input automator.
```
Comment (CleanMachine1) [inline pages/linux/ydotool.md:3]: Isn't this software more of a keyboard "emulator" for entering keys and such through commands?

If so it may be better to communicate this to the user.

```suggestion
> Control keyboard and mouse inputs via commands.
```

File: pages/linux/ydotool.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://github.com/ReimuNotMoe/ydotool>.
+
+- Left click with a mouse:
```
Comment (CleanMachine1) [inline pages/linux/ydotool.md:6]:
```suggestion
- Perform a left click input:
```

File: pages/linux/ydotool.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`ydotool click 0xC0`
+
+- Right click with a mouse:
```
Comment (CleanMachine1) [inline pages/linux/ydotool.md:10]:
```suggestion
- Perform a right click input:
```

File: pages/linux/ydotool.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`ydotool click 0xC1`
+
+- Send the keys Alt+F4:
```
Comment (CleanMachine1) [inline pages/linux/ydotool.md:14]: not too sure with this one, some feedback would be good.
```suggestion
- Input Alt and F4:
```

File: pages/linux/ydotool.md (hunk 1 above)
Comment (cyqsimon) [reply pages/linux/ydotool.md:3]: I feel like "Display server agnostic" is kind of important, because `ydotool` derives its name from `xdotool`, which specifically targets X11.

File: pages/linux/ydotool.md (hunk 4 above)
Comment (cyqsimon) [reply pages/linux/ydotool.md:14]: The original is better because "Input Alt and F4" is more ambiguous on the release order. The key sequence here is "alt down", "f4 down", "f4 up", "alt up".

File: pages/linux/ydotool.md (hunk 1 above)
Comment (Managor) [reply pages/linux/ydotool.md:3]: Yeah, it's the reason why I chose that wording. If there was no succession line, I would have chosen a different description.

File: pages/linux/ydotool.md (hunk 4 above)
Comment (Managor) [reply pages/linux/ydotool.md]: I adjusted it a bit while keeping Alt+F4

File: pages/linux/ydotool.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/linux/ydotool.md:3]: I only suggested it since it makes the most sense to me, I have no idea what a server agnostic is. Can we come up a wording which makes sense to both newbs and people who the software.

Comment (Managor) [reply pages/linux/ydotool.md:3]: "Display server" agnostic. `xdotool` connects to X11 directly to input keys and mouse events whereas `ydotool` creates a virtual input device. Your comment gave me an idea how to reword this. Just a moment.

**Comment by Managor** [comment]
Last minute changes

**Review by cyqsimon — APPROVED** [review]
`> Control keyboard and mouse inputs via commands in a way that is display server agnostic.`

TBH I find this a bit wordy for my taste but it's not a deal-breaker or anything.

**Comment by cyqsimon** [comment]
Thanks!
